Sources/Foundation/Process.swift

```diff
@@ -226,9 +226,9 @@ private func quoteWindowsCommandLine(_ commandLine: [String]) -> String {
 #endif
 
 open class Process: NSObject, @unchecked Sendable {
+    static let once = Mutex(false)
+    
     private static func setup() {
-        let once = Mutex(false)
-        
         once.withLock {
```

Comment:
Not sure if this needs a `Self.` in front of once, but if it builds it looks good to me!

Reply:
Since the function is also static it is inferred.